releases/release-1.20/release-notes-draft.md

```diff
@@ -396,24 +493,39 @@
 
 ### Other (Cleanup or Flake)
 
```

Comment:
We're removing the documentation for the cloud-controller-manager skeleton binary (which doesn't actually integrate with any provider).

SIG API Machinery, is that correct? Slack discussion starts at https://kubernetes.slack.com/archives/C0EG7JC6T/p1606929363248600

Reply:
I'm going to follow the conversation -- thanks for flagging Tim!